client/app/visualizations/counter/Renderer.jsx

```diff
@@ -0,0 +1,74 @@
+import { isFinite } from 'lodash';
+import React, { useState, useEffect } from 'react';
+import cx from 'classnames';
+import resizeObserver from '@/services/resizeObserver';
+import { RendererPropTypes } from '@/visualizations';
+
+import { getCounterData } from './utils';
+
+import './render.less';
+
+function getCounterStyles(scale) {
+  return {
+    OTransform: `scale(${scale})`,
+    MsTransform: `scale(${scale})`,
```

Comment:
I believe this should be `msTransform`.

In any case, wdyt of using `modernizr-prefixed` or a simple custom util here instead?

Reply:
Yeah, you're right - `-ms-` is the only exception :man_facepalming: Also, probably I'll remove `-o-` rule (Opera 12 is in fact dead)

Reply:
`modernizr` seems to be an overkill here :thinking: I'll check if we're using something similar elsewhere - I'll create a custom utility